diff -up dnf5-5.1.13/bindings/libdnf5/comps.i.omv~ dnf5-5.1.13/bindings/libdnf5/comps.i
--- dnf5-5.1.13/bindings/libdnf5/comps.i.omv~	2024-03-04 18:56:46.302183010 +0100
+++ dnf5-5.1.13/bindings/libdnf5/comps.i	2024-03-04 18:57:56.739034195 +0100
@@ -38,6 +38,7 @@
     #include "libdnf5/comps/environment/sack.hpp"
     #include "libdnf5/comps/comps.hpp"
     #include "libdnf5/repo/repo.hpp"
+    #include "libdnf5/repo/repo_query.hpp"
 %}
 
 #define CV __perl_CV
diff -up dnf5-5.1.13/dnf5-plugins/automatic_plugin/automatic.cpp.omv~ dnf5-5.1.13/dnf5-plugins/automatic_plugin/automatic.cpp
--- dnf5-5.1.13/dnf5-plugins/automatic_plugin/automatic.cpp.omv~	2024-03-04 18:50:09.201056577 +0100
+++ dnf5-5.1.13/dnf5-plugins/automatic_plugin/automatic.cpp	2024-03-04 18:50:18.929191083 +0100
@@ -51,7 +51,7 @@ static void random_wait(int max_value) {
     std::mt19937 rng(rd());
     std::uniform_int_distribution<int> distribution(0, max_value);
 
-    sleep(distribution(rng));
+    sleep(static_cast<unsigned int>(distribution(rng)));
 }
 
 static bool reboot_needed(libdnf5::Base & base, const libdnf5::base::Transaction & transaction) {
diff -up dnf5-5.1.13/dnf5-plugins/automatic_plugin/emitters.hpp.omv~ dnf5-5.1.13/dnf5-plugins/automatic_plugin/emitters.hpp
--- dnf5-5.1.13/dnf5-plugins/automatic_plugin/emitters.hpp.omv~	2024-03-04 18:48:02.647314639 +0100
+++ dnf5-5.1.13/dnf5-plugins/automatic_plugin/emitters.hpp	2024-03-04 18:49:53.320835771 +0100
@@ -41,6 +41,7 @@ public:
           transaction(transaction),
           output_stream(output_stream),
           success(success) {}
+    virtual ~Emitter() {}
 
     /// Notify the user about the status of dnf-automatic run.
     virtual void notify() = 0;
