diff -up dnf5-5.1.9/include/libdnf5-cli/output/provides.hpp.omv~ dnf5-5.1.9/include/libdnf5-cli/output/provides.hpp
--- dnf5-5.1.9/include/libdnf5-cli/output/provides.hpp.omv~	2024-01-10 16:26:06.565305035 +0000
+++ dnf5-5.1.9/include/libdnf5-cli/output/provides.hpp	2024-01-10 16:26:14.145295601 +0000
@@ -31,7 +31,7 @@ namespace libdnf5::cli::output {
 
 enum ProvidesMatchedBy : int { NO_MATCH = 0, PROVIDES = 1, FILENAME = 2, BINARY = 3 };
 
-static void add_line_into_provides_table(struct libscols_table * table, const char * key, const char * value) {
+static inline void add_line_into_provides_table(struct libscols_table * table, const char * key, const char * value) {
     struct libscols_line * ln = scols_table_new_line(table, nullptr);
     scols_line_set_data(ln, 0, key);
     scols_line_set_data(ln, 1, value);
