diff -up dnf5-5.3.0.0/libdnf5/conf/vars.cpp.2~ dnf5-5.3.0.0/libdnf5/conf/vars.cpp
--- dnf5-5.3.0.0/libdnf5/conf/vars.cpp.2~	2025-11-04 15:00:17.000000000 +0100
+++ dnf5-5.3.0.0/libdnf5/conf/vars.cpp	2025-12-13 03:14:06.829902883 +0100
@@ -55,6 +55,7 @@ namespace libdnf5 {
 static const std::unordered_set<std::string> READ_ONLY_VARIABLES = {};
 
 static constexpr const char * DISTROVERPKGS[] = {
+    "distro-release-common",
     "system-release(releasever)",
     "system-release",
     "distribution-release(releasever)",
